package.json

```diff
@@ -14,7 +14,7 @@
     "@metamask/jazzicon": "2.0.0",
     "@oasisprotocol/client": "0.1.0-alpha7",
     "@oasisprotocol/client-ext-utils": "0.1.0-alpha3",
-    "@oasisprotocol/client-rt": "0.2.0-alpha8",
+    "@oasisprotocol/client-rt": "0.2.0-alpha10",
```

Comment:
we should commit the corresponding yarn.lock changes too

Reply:
Ok , thanks for the suggestion

Reply:
i commit yarn in this [commit ](https://github.com/oasisprotocol/oasis-wallet-ext/pull/168/commits/6b61dcc23be6062b1fc6e14c3a166fc6c632cea0)

Reply:
got it, thanks